src/olympia/migrations/952-correct-index-theme-user-counts.sql

```diff
@@ -0,0 +1,5 @@
+-- Drop old index first
+DROP INDEX `addon_date_idx` ON `theme_user_counts`;
```

Comment:
That will fail in environments where the index does not exist... I fear you need to write this in a more defensive way.